### DISCUSSION TITLE
Anserini Retrieval latency question - Mono thread/CPU ?

### DISCUSSION THREAD

**Discussion by PrithivirajDamodaran** [castorini/anserini#2376] [Q&A]
I am trying to reproduce the retrieval latency and index size for splade v2 reported in the Wacky Weights paper. 

How's the retrieval latency calculated was it just a mean latency of of all queries on the msmarco dev set done using mono-thread no batching or threading and batched queries ?

Thanks

**Comment by lintool** [comment]
IIRC @JMMackenzie performed the experiments, so he'll have to confirm - but I assume single thread no batching.
